get_together/context_processors.py

```diff
@@ -0,0 +1,5 @@
+from django.conf import settings
+
+
```

Comment:
stray newline here.

Reply:
This is actually a pep8 convention 😛 Since we are considering using a code formatter, I think we should keep the basic conventions to avoid extra diffs later. Thoughts?
https://www.python.org/dev/peps/pep-0008/#blank-lines